source/hud/Hotbar.cpp

```diff
@@ -29,11 +29,11 @@ Hotbar::Hotbar(Inventory &inventory, Widget *parent) : Widget(182, 22, parent),
 	}
 }
 
-void Hotbar::onEvent(const SDL_Event &event) {
-	if (event.type == SDL_MOUSEWHEEL) {
-		if (event.wheel.y == -1)
+void Hotbar::onEvent(const sf::Event &event) {
+	if (event.type == sf::Event::MouseWheelScrolled) {
+		if (int(event.mouseWheelScroll.delta) == -1)
 			m_cursorPos = (m_cursorPos + 1) % 9;
-		else if (event.wheel.y == 1)
+		else if (int(event.mouseWheelScroll.delta) == 1)
```

Comment:
Why do you have to cast `delta` to `int`?

Reply:
Because SFML gives the delta as a float (apparently some scroll wheels support that). It might be no problem for most normal mice, but I thought making comparison between float and int explicit is better.

Reply:
Hmm should we instead do `else if (event.mouseWheelScroll.delta > 0)`?
This would be better because if you scroll really fast I think it can be higher than 1.
It still doesn't scroll by more than 1 in a frame, but at least it does something.